src/lib/kw_config_loader.sh

```diff
@@ -358,8 +358,11 @@ function parse_configuration()
   if [ ! -f "$config_path" ]; then
     return 22 # 22 means Invalid argument - EINVAL
   fi
+
+  # @see https://stackoverflow.com/questions/12916352/shell-script-read-missing-last-line
```

Comment:
Maybe summarize the explanation in this link in the commit message.

Reply:
Agreed. If you think that a comment is necessary here (I see that is not straigthfoward to understand) try to explain it in the comment itself in a succinct way, if possible.